packages/webpack/lib/plugins/log.js

```diff
@@ -38,7 +38,13 @@ const formatWarning = (name, duration, assets, isRebuild) => {
 };
 
 const formatSuccess = (name, duration, assets, isRebuild) => {
-  const message = `bundling '${name}' finished after ${duration}`;
+  const totalSize = Object.values(assets).reduce(
+    (sum, asset) => sum + asset.size,
+    0
+  );
+  const message = `bundling '${name}' finished after ${duration} (${prettyBytes(
```

Comment:
I would probably print here something like `bundling .. finished after .. combined size of all assets: ...`

Reply:
I just used the same format as the other assets:
```
hops-demo:info bundling 'node' finished after 1m 5.1s (14.7 MB)
- server.js (14.5 MB)
- stats.json (187 kB)
```

Reply:
Ok, let's leave it like that then. I just thought it might be irritating if there's no explanation what this number means.